packages/js/src/ui/components/Popover/PopoverContent.tsx

```diff
@@ -0,0 +1,63 @@
+import { onCleanup, onMount, ParentComponent, Show } from 'solid-js';
+import { Portal } from 'solid-js/web';
+import { useStyle } from '../../helpers';
+import { usePopover } from './Popover';
+import { useFloating } from 'solid-floating-ui';
+
+import { autoUpdate, offset, flip, shift } from '@floating-ui/dom';
+export const PopoverContent: ParentComponent = (props) => {
+  const style = useStyle();
+  const { setContentRef, opened, targetRef, onClose, contentRef } = usePopover();
+
+  const position = useFloating(targetRef, contentRef, {
+    placement: 'bottom-end',
+    whileElementsMounted: autoUpdate,
+
+    middleware: [offset(10), flip(), shift()],
+  });
+
+  const handleClickOutside = (e: any) => {
```

Comment:
Doesn't floating ui handle this ?

Reply:
No, I checked and it doesnt support it, they have this feature in their framework packages (react, vue)